serendipity_event_dejure/serendipity_event_dejure.php

```diff
@@ -213,9 +241,10 @@ function djo_vernetzen_ueber_cache(&$ausgangstext) {
         global $serendipity;
 
     	$schluessel = md5($ausgangstext);
-    	$rueckgabe = serendipity_db_query("SELECT cache FROM {$serendipity['dbPrefix']}dejure WHERE ckey = '" . $schluessel . "'", true, 'assoc');
+        $rueckgabe = serendipity_db_query("SELECT cache FROM {$serendipity['dbPrefix']}dejure
+                                           WHERE ckey = '".$schluessel."' AND last_update > " . (time()-86400*CACHE_VORHALT), true, 'assoc');
     	if (!empty($rueckgabe['cache']) && $rueckgabe['cache'] == "<!-- idem -->") {
-		return $ausgangstext;
+		    return $ausgangstext;
```

Comment:
Amen, brother. :)

I plan to update the plugin some more as soon as I have some free time on my hands. I've got a 1.6-beta from upstream, and I'll add some new functions the other plugins already got, too.

After that I'll have a look at the variable names.